src/i18n/locales/en/translation.json

```diff
@@ -148,6 +148,16 @@
       "no_transactions": "No transactions yet."
     }
   },
+  "accounts": {
+    "export": {
+      "name": "Export account details",
+      "waiting": "waiting for LndHub data...",
+      "your_ln": "Your Lightning Address:",
```

Comment:
I think we should not skip the `address` there.
your `your_ln_address` is more description/correct, imo?